sdk/cognitivelanguage/Azure.AI.Language.Conversations/README.md

```diff
@@ -302,28 +301,27 @@ var data = new
 
 Response response = client.AnalyzeConversation(RequestContent.Create(data));
 
-using JsonDocument result = JsonDocument.Parse(response.ContentStream);
-JsonElement conversationalTaskResult = result.RootElement;
-JsonElement orchestrationPrediction = conversationalTaskResult.GetProperty("result").GetProperty("prediction");
+dynamic conversationalTaskResult = response.Content.ToDynamicFromJson();
+dynamic orchestrationPrediction = conversationalTaskResult.result.prediction;
 ```
 
 #### Question Answering prediction
 
 If your conversation was analyzed by Question Answering, it will include an intent - perhaps even the top intent - from which you can retrieve answers:
 
 ```C# Snippet:ConversationAnalysis_AnalyzeConversationOrchestrationPredictionQnA
-string respondingProjectName = orchestrationPrediction.GetProperty("topIntent").GetString();
-JsonElement targetIntentResult = orchestrationPrediction.GetProperty("intents").GetProperty(respondingProjectName);
+string respondingProjectName = orchestrationPrediction.topIntent;
+dynamic targetIntentResult = orchestrationPrediction.intents[respondingProjectName];
 
-if (targetIntentResult.GetProperty("targetProjectKind").GetString() == "QuestionAnswering")
+if (targetIntentResult.targetProjectKind == "QuestionAnswering")
 {
     Console.WriteLine($"Top intent: {respondingProjectName}");
 
-    JsonElement questionAnsweringResponse = targetIntentResult.GetProperty("result");
+    dynamic questionAnsweringResponse = targetIntentResult.result;
     Console.WriteLine($"Question Answering Response:");
-    foreach (JsonElement answer in questionAnsweringResponse.GetProperty("answers").EnumerateArray())
+    foreach (dynamic answer in questionAnsweringResponse.answers)
     {
-        Console.WriteLine(answer.GetProperty("answer").GetString());
+        Console.WriteLine(answer.answer?.ToString());
```

Comment:
Consider:
```suggestion
        Console.WriteLine($"{answer.answer}");
```

Reply:
That's actually more expensive and ends up calling the same thing anyway. It allocates an `IFormattableString` implementation, scans text for a myriad of interfaces that can change formatting, and ultimately falls back to calling `object.ToString()`: https://www.codeproject.com/Articles/6533/Custom-String-Formatting-in-NET (it's old but more or less correct).

That said, the compiler may optimize some of this away, but I still prefer calling `ToString()` when I know exactly what I need and format doesn't need to vary by locale or be passed format specifiers.